src/components/include/utils/threads/thread.h

```diff
@@ -90,7 +98,12 @@ void sleep(uint32_t ms);
 Thread* CreateThread(const char* name, ThreadDelegate* delegate);
 void DeleteThread(Thread* thread);
 
+#ifdef QT_PORT
+class Thread : public QObject {
```

Comment:
@nk0leg I don't see any signals or slots in this case.
For MOC usage in source you have no need to inherit QObject.